tools/update_quest_data.py

```
@@ -0,0 +1,439 @@
+    region = args.region.upper()
+
+    if args.missions_only:
+        update_master_missions(region)
+    elif args.quests_only:
+        if args.source == "chaldea":
+            update_quest_enemies_from_chaldea(region)
+        elif args.source == "atlas":
+            update_quest_enemies(region)
+        else:
+            # auto: 国服默认用 Chaldea，其他用 Atlas
+            if region == "CN":
```

Ruff (0.15.11)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)